model-optimizer/extensions/ops/bucketize.py

```diff
@@ -16,6 +16,7 @@
 
 import numpy as np
 
+from mo.front.extractor import get_boolean_attr
```

Comment:
I think that the name of the function is misleading. The function converts boolean attr to string. May be call it `bool_to_string`?

Reply:
Ok, done.